<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1074</width>
    <height>734</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PWM-Control</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="PlotWidget" name="pwm3View" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="PlotWidget" name="pwm1View" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="PlotWidget" name="pwm2View" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <property name="sizeConstraint">
       <enum>QLayout::SetMaximumSize</enum>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>100</height>
         </size>
        </property>
        <property name="baseSize">
         <size>
          <width>0</width>
          <height>105</height>
         </size>
        </property>
        <property name="title">
         <string>Control</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>60</y>
           <width>49</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Duty 1:</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_2">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>90</y>
           <width>49</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Duty 2:</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_3">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>120</y>
           <width>49</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Duty 3:</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="edit_duty1">
         <property name="geometry">
          <rect>
           <x>70</x>
           <y>60</y>
           <width>71</width>
           <height>21</height>
          </rect>
         </property>
        </widget>
        <widget class="QSlider" name="sl_pwm1">
         <property name="geometry">
          <rect>
           <x>210</x>
           <y>60</y>
           <width>160</width>
           <height>20</height>
          </rect>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
        <widget class="QPushButton" name="pb_Test">
         <property name="geometry">
          <rect>
           <x>960</x>
           <y>80</y>
           <width>84</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>Exit</string>
         </property>
        </widget>
        <widget class="QSlider" name="sl_pwm3">
         <property name="geometry">
          <rect>
           <x>210</x>
           <y>120</y>
           <width>160</width>
           <height>20</height>
          </rect>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
        <widget class="QSlider" name="sl_pwm2">
         <property name="geometry">
          <rect>
           <x>210</x>
           <y>90</y>
           <width>160</width>
           <height>20</height>
          </rect>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
        <widget class="QLineEdit" name="edit_duty2">
         <property name="geometry">
          <rect>
           <x>70</x>
           <y>90</y>
           <width>71</width>
           <height>21</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="edit_duty3">
         <property name="geometry">
          <rect>
           <x>70</x>
           <y>120</y>
           <width>71</width>
           <height>21</height>
          </rect>
         </property>
        </widget>
        <widget class="QRadioButton" name="lockButton">
         <property name="geometry">
          <rect>
           <x>390</x>
           <y>90</y>
           <width>93</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>Lock</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="edit_freq">
         <property name="geometry">
          <rect>
           <x>70</x>
           <y>30</y>
           <width>71</width>
           <height>21</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_4">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>30</y>
           <width>49</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Freq:</string>
         </property>
        </widget>
        <widget class="QComboBox" name="comboBox">
         <property name="geometry">
          <rect>
           <x>839</x>
           <y>30</y>
           <width>201</width>
           <height>21</height>
          </rect>
         </property>
        </widget>
        <widget class="QFrame" name="frame">
         <property name="geometry">
          <rect>
           <x>200</x>
           <y>20</y>
           <width>571</width>
           <height>30</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <property name="lineWidth">
          <number>0</number>
         </property>
         <widget class="QRadioButton" name="f1Button">
          <property name="geometry">
           <rect>
            <x>190</x>
            <y>10</y>
            <width>93</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>1 Hz</string>
          </property>
         </widget>
         <widget class="QRadioButton" name="f10Button">
          <property name="geometry">
           <rect>
            <x>280</x>
            <y>10</y>
            <width>93</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>10 Hz</string>
          </property>
         </widget>
         <widget class="QSlider" name="freqSlider">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>10</y>
            <width>160</width>
            <height>20</height>
           </rect>
          </property>
          <property name="maximum">
           <number>999</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
         <widget class="QRadioButton" name="f100Button">
          <property name="geometry">
           <rect>
            <x>370</x>
            <y>10</y>
            <width>93</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>100 Hz</string>
          </property>
         </widget>
         <widget class="QRadioButton" name="f1000Button">
          <property name="geometry">
           <rect>
            <x>470</x>
            <y>10</y>
            <width>93</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>1 KHz</string>
          </property>
         </widget>
        </widget>
        <widget class="QLabel" name="label_5">
         <property name="geometry">
          <rect>
           <x>150</x>
           <y>63</y>
           <width>49</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>%</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_6">
         <property name="geometry">
          <rect>
           <x>150</x>
           <y>93</y>
           <width>49</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>%</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_7">
         <property name="geometry">
          <rect>
           <x>150</x>
           <y>123</y>
           <width>49</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>%</string>
         </property>
        </widget>
        <widget class="QLabel" name="freqLabel">
         <property name="geometry">
          <rect>
           <x>150</x>
           <y>33</y>
           <width>49</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Hz</string>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1074</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionTest">
   <property name="text">
    <string>Test</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QWidget</extends>
   <header location="global">pyqtgraph</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
